### PR TITLE
Satsuma2-20220304 with SATSUMA2_PATH env var

### PR DESCRIPTION
**Pull request by orbsmiv**
For INC1504383 - `Satsuma2-20220304-GCC-11.3.0.eb`

Copy Satsuma easyconf file from upstream but add SATSUMA2_PATH env var

* [x] Assigned to reviewer

Default:
* [ ] EL8-icelake
* [ ] EL8-cascadelake

### CODE REVIEW

File: easyconfigs/s/Satsuma2/Satsuma2-20220304-GCC-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    'c80755f8942f3a0c9a2598b2a1a6319f508956bc29ccdfdfb9044ec8c50cd691',  # Satsuma2-20220304_fix-GCC-11.patch
+]
+
+builddependencies = [('CMake', '3.23.1')]
+
+# Define SATSUMA2_PATH as per:
+# https://github.com/bioinfologics/satsuma2/blob/37c5f386819614cd3ce96016b423ddc4df1d86ec/README.md#running-satsuma2
+modextravars = {
+    'SATSUMA2_PATH': '%(installdir)s/bin',
+}
```
Comment (orbsmiv) [inline easyconfigs/s/Satsuma2/Satsuma2-20220304-GCC-11.3.0.eb:28]: This is the modified code when compared with the upstream easyconf

Comment (branfosj) [reply easyconfigs/s/Satsuma2/Satsuma2-20220304-GCC-11.3.0.eb:28]: Upstream this change and we'll install from the PR.

**Comment by orbsmiv** [comment]
Closing and will install from upstream PR instead.
(See https://github.com/bear-rsg/bear-eb/pull/410#discussion_r1664034465)

**Comment by orbsmiv** [comment]
See https://github.com/easybuilders/easybuild-easyconfigs/pull/20957

**Pull request closed without merging**